packages/api/src/routes/helpers/provider-route-helper.ts

```diff
@@ -20,7 +21,10 @@ export async function getProviderDataForType<T>(
   const date = getDateOrFail(req);
   const params: DAPIParams = {
     timezoneId: getTimezoneIdFrom("query", req).optional(),
+    xTenoviApiKey: getTenoviApiKeyFrom("headers", req).optional(),
+    xTenoviClientName: getTenoviClientNameFrom("headers", req).optional(),
```

Comment:
I think we should find another place for this.. these are provider-specific and this code is generic.

I'd rather pass something like this to `getXXXData` than having this here:

```ts
type rawParams = {
  query: Record<string, string>;
  headers: Record<string, string>;
}
```